### PR TITLE
Added ability to aim the cue by dragging the mouse

### PR DESCRIPTION
**Pull request by ENiGMA9**
This solves #62

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -801,7 +803,8 @@
    // event handlers
    var mouseClick = function(e)
    {
-       var cue_ball = ballList.balls[0];
+       //Obsolete now but I leave it here for easy modifications
+      /* var cue_ball = ballList.balls[0];
```
Comment (sleddog) [inline index.html]: @ENiGMA9 if this isn't needed feel free to clean up.  Github keeps the history nicely :)

**Comment by ENiGMA9** [comment]
Thanks, I did as suggested, now about the click thing, currently works anywhere in the canvas not only on the ghost ball, would you like me to change this?

**Comment by sleddog** [comment]
@ENiGMA9 Sure, if you can improve it give it a shot (pun intended).  Looking for ideas to improve how the user aims the next shot.

**Comment by ENiGMA9** [comment]
@sleddog I worked a bit last night but didn't manage easily to pull this off, I meant to commit to this PR but I didn't do it in time and I still have to work on it as it's not functional right now, although I would say it's a bit better like it is now, control-wise? I feel like 8ball pool from miniclip is good on controls, if you want to take inspiration from that.

**Comment by sleddog** [comment]
@ENiGMA9 8ball pool from miniclip is rock solid.  If we could build a game that is close to that, then wow!  For now I'm happy with the aiming improvements you added.  If you have more ideas for improvement let me know.  Thanks again.

**Pull request merged**